src/assets/browser/app/src/composit/mod.rs

```diff
@@ -6,12 +6,10 @@ mod componentset;
 mod landscape;
 mod leaf;
 pub mod source;
-mod stage;
 mod stick;
 mod stickmanager;
-mod zoom;
-mod position;
 mod wrapping;
+mod stage;
```

Comment:
(Whole file) separate "stage" position management into own module. Only lightly coupled with rest of package and easier to factor when methods are private to a small module.